docs/basics/test-config-annotation/test-annotation.md

```diff
@@ -149,10 +153,12 @@ public class TextInputTest extends TestBase {
     }
 ```
 ## Sauce Labs REST API
-You can manage your tests more effectively from your Dashboard and Archives with annotations. The Sauce Labs REST API includes an [update_job](/dev/api/jobs) method that you can use to set a name, tags, pass/fail status, and custom data for your test after it runs. To automate test annotation with this method, you'll want to create a simple set of functions to perform the put request for you. We've developed a [Java library](https://github.com/saucelabs/saucerest-java) to do just that, with examples for [Python](https://gist.github.com/1644439) and [Ruby](https://gist.github.com/DylanLacey/5218959) on GitHub.
+You can manage your tests more effectively from your Dashboard and Archives with annotations. The Sauce Labs REST API includes an [update_job](/dev/api/jobs/#update-a-job) method that you can use to set a name, tags, pass/fail status, and custom data for your test after it runs. To automate test annotation with this method, you'll want to create a simple set of functions to perform the put request for you. We've developed a [Java library](https://github.com/saucelabs/saucerest-java) to do just that, with examples for [Python](https://gist.github.com/1644439) and [Ruby](https://gist.github.com/DylanLacey/5218959) on GitHub.
 
-:::note
-Adding Pass/Fail Status and Build Numbers to Test Results with Frameworks<br/>
+:::note RDC Jobs
+The [update_job](/dev/api/jobs/#update-a-job) method is not supported for RDC. To update a job for real devices, use the [Selenium JavaScript Executor](/basics/test-config-annotation/test-annotation/#selenium-javascript-executor).
```

Comment:
See my remark about the update job API, maybe remove the above note and mention that it **can't** be used for RDC

Reply:
Removed.